docs/get-started/build/quickstart/deploy.mdx

```diff
@@ -0,0 +1,253 @@
+`https://rpc.sepolia.linea.build`.
+
+Public endpoints such as this one are rate limited, and if you intend to deploy your app for public
+use, you'll need an endpoint better equipped to handle the load. We recommend heading to the 
+MetaMask Developer Dashboard to get a private RPC endpoint ([instructions](https://docs.metamask.io/developer-tools/dashboard/get-started/create-api/)).
+
+You can also find alternative public and private RPC endpoints on our [node providers page](../../tooling/node-providers/index.mdx).
+
+#### Update `foundry.toml`
+
+Add the RPC endpoint to the `foundry.toml` file:
+
+```toml
+[profile.rpc-endpoints]
+sepolia = "https://rpc.sepolia.linea.build"
+```
+
+### Store your private key
+
+Deploying a contract requires an account's private key, which means you need to either provide it in
+the command line when running the deployment command (not recommended) or store it securely and 
+access it with alternative methods. See the [MetaMask support guide to finding your private key](https://support.metamask.io/managing-my-wallet/secret-recovery-phrase-and-private-keys/how-to-export-an-accounts-private-key/).
+
+:::tip[Private key security best practices]
+
+We recommend [creating a new account in MetaMask](https://support.metamask.io/managing-my-wallet/accounts-and-addresses/how-to-add-accounts-in-your-wallet/)
+specifically for this project, and [naming it](https://support.metamask.io/managing-my-wallet/accounts-and-addresses/how-do-i-change-my-account-name-/)
+something appropriate, such as "linea-project".
+
+Even though we're working on a testnet, it's best to behave as if real assets were at stake, and
+work in a sandboxed context with a somewhat expendable account.
+
+:::
+
+A standard approach is to store your private key as an environment variable in a `.env` file that 
+is only stored locally (i.e. listed in `.gitignore` file). However, we're going to use Foundry's
+keystore system to encrypt and store the private key, and then access it with a password. This 
+ensures that you won't accidentally expose your private key—for example, in a GitHub repository.
+
+To do this, we'll use Foundry's [`cast`](https://book.getfoundry.sh/cast/) CLI tool; specifically,
+the [`cast wallet import`](https://book.getfoundry.sh/reference/cast/cast-wallet-import) command.
+
+```bash
+cast wallet import --interactive test-account
+```
+
+The `--interactive` flag means that you'll be prompted to enter your private key in the terminal,
+followed by a corresponding password. `test-account` is the name you want to give the account in 
+the keystore; choose whatever is suitable for your purposes here.
+
+When you enter the private key and then your password, you'll get confirmation that it has been 
+stored successfully:
+
+```
+`test-account` keystore was saved successfully. Address: <YOUR_PUBLIC_ADDRESS>
+```
+
+Run `cast wallet list` to see accounts in your keystore.
+
+### Deploy
+
+#### Adjust Solidity version
```

Comment:
This Vale rule in particular should, potentially, be removed. We will be using proper nouns in headings... A lot lmao

Reply:
Surprised/disappointed it wasn't already on an ignore list. Added to my tracker of things to add next time!